tensorboard/plugins/hparams/hparams_demo.py

```diff
@@ -31,8 +35,9 @@
 import shutil
 
 from six.moves import xrange  # pylint: disable=redefined-builtin
-import tensorflow as tf
-from tensorflow import flags
+import tensorflow.compat.v1 as tf
+from absl import flags
+from absl import app
```

Comment:
please add absl dependencies on the BUILD file.

Reply:
Done.